oodeel/eval/metrics.py

```diff
@@ -111,13 +112,10 @@ def bench_metrics(
                 ).groups()
                 thr = int(thr)
                 count_1, count_2 = locals()[count_1_str], locals()[count_2_str]
-                for i, c2 in enumerate(count_2):
-                    if (count_2_str in ["fpr", "tpr"] and c2 < thr / 100) or (
-                        count_2_str in ["tnr", "fnr"] and c2 > thr / 100
-                    ):
-                        ind = i
-                        break
-                metrics_dict[metric] = count_1[ind]
+                interp = interp1d(
```

Comment:
Does this change is strictly identical to the previous computation when the ROC is "normal"?